config.go

```diff
@@ -41,11 +41,10 @@ const (
 	pauseBinRelativePath        = "bin/pause"
 )
 
-var defaultShimPath = "/usr/local/libexec/cc-shim"
-
 const (
-	qemuLite = "qemu-lite"
-	qemu     = "qemu"
+	qemuLite   = "qemu-lite"
```

Comment:
guess we'll let you get away with the mixed in whitespace change @jodh-intel  (threw my review eyes out for a moment ;-) )